packages/gasket-plugin-middleware/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@gasket/plugin-middleware",
+  "version": "1.0.0",
+  "description": "Handles common server engine setups for routing and executing lifecycles.",
+  "main": "lib",
+  "scripts": {
+
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage"
```

Comment:
Can you add `typecheck` script and dependencies? This will surface a few type issues in your PR.